packages/millicast-sdk/src/utils/Codecs.ts

```diff
@@ -64,28 +38,43 @@ export const DOLBY_SEI_TIMESTAMP_UUID = '9a21f3be-31f0-4b78-b0be-c7f7dbb97250'
 // When the SDK inserts its own timecode into the unregistered block, it uses this identifier
 export const DOLBY_SDK_TIMESTAMP_UUID = 'd40e38ea-d419-4c62-94ed-20ac37b4e4fa'
 
+export interface SeiMetadata {
+  seiPicTimingTimeCodeArray: SeiPicTimingTimeCode[]
```

Comment:
Shouldn't this be optional? As it is being set in a switch case

Reply:
I canged this to optional.